helm/hpcc/templates/thor.yaml

```diff
@@ -319,6 +319,12 @@ data:
               job: "_HPCC_JOBNAME_"
 {{- end -}}
 
+{{- if empty $.Values.dali -}}
```

Comment:
observation: Repeated lots of times.  I wonder if it is worth commoning up.  Probably not given the complication of helm functions.  Would it be better as a global consistency check?

Reply:
I will look at both options.  Do you know if there is a way to guarantee that a global consistency check is run before any of the individual components templates?  I suppose I could create a global conistency check function that is called at the top of each template, but would check everything and fail if something is wrong... I THINK I can cache the fact that it was already run and skip after the first time by writing a value.  Would need to play with it a bit.

Reply:
I wouldn't spend too long on it... the code is ok as it stands.

Reply:
see my comment/question above, re. could check go in daliArg helper with some passed in context (.)